packages/vue-cli-plugin-vuetify/generator/tools/fonts.js

```diff
@@ -67,8 +67,26 @@ function addLinks (api, iconFont) {
   })
 }
 
+function addPlugin(api, opts) {
+  opts.hasTS = api.hasPlugin('typescript')
+  const ext = opts.hasTS ? 'ts' : 'js'
+
+  api.injectImports(api.entryFile, `import { loadFonts } from './plugins/webfontloader'`)
+  
+  api.extendPackage({
+    dependencies: {
+      'webfontloader': '^1.0.0',
```

Comment:
```suggestion
      'webfontloader': '^1.6.28',
```

Reply:
It will resolve to that version in the yarn.lock file:
<img width="1106" alt="Screen Shot 2021-07-14 at 8 44 35 PM" src="https://user-images.githubusercontent.com/24883328/125726043-19bb1da8-81b5-4dfa-bafc-dc6d4b8c708f.png">

Reply:
I'm tracking. If you feel it's a menial change, you can resolve the discussion.